### ISSUE TITLE
Cannot set custom session ID since v2.4.0

### ISSUE THREAD

**Issue by jandamm** [TelemetryDeck/SwiftSDK#192]
With v2.3.0 I was able to keep a reference to `TelemetryClient.Config` and update the `sessionID`.

With v2.4.0 onwards `TelemetryClient.Config` is a struct so I cannot keep a reference and cannot update the `sessionID` or other values of the config. There also isn' a way to get to the `Config` in any other way.

The comments in v2.4.1 still say:
```
/// Use an instance of this class to specify settings for TelemetryManager. If these settings change during the course of
/// your runtime, it might be a good idea to hold on to the instance and update it as needed. TelemetryManager's behaviour
/// will update as well.
```

Is there a way to still set a custom `sessionID` I didn't find?

**Comment by Jeehut** [comment]
@jandamm Thank you for reporting this and sorry for the inconvenience. This was introduced as part of our attempt to migrate our library to Swift 6 (see our [blog article](https://telemetrydeck.com/blog/migrating-to-swift-6/)). It seems truning the config to a `struct` was the wrong approach. I've reverted that in #206. Once it's reviewed I'll make a new version release which should get everything back to working state again! Stay tuned.

**Comment by Jeehut** [comment]
@jandamm I'm happy to inform you that the fix is now part of the [new SDK release 2.6.0](https://github.com/TelemetryDeck/SwiftSDK/releases/tag/2.6.0). Please give it a try and let me know if you run into any issues! Thank you again for reporting this. 🙏

**Comment by jandamm** [comment]
Thanks, looks good 👍
